lib/telemetry_metrics_mnesia/event_handler.ex

```diff
@@ -34,11 +37,27 @@ defmodule TelemetryMetricsMnesia.EventHandler do
           :telemetry.event_metadata(),
           metrics()
         ) :: :ok | {:error, any()}
-  def handle_event(event, measurements, metadata, _metrics) do
+  def handle_event(event, measurements, metadata, metrics) do
     # Logger.debug(
-    #   ~c"handle_event(#{inspect(event)}, #{inspect(measurements)}, #{inspect(metadata)}, _)"
+    #   ~c"handle_event(#{inspect(event)}, #{inspect(measurements)}, #{inspect(metadata)}, #{inspect(metrics)})"
     # )
 
-    Db.write_event(event, measurements, metadata)
+    GenServer.call(__MODULE__, {:event, event, measurements, metadata, metrics})
+
+    :ok
+  end
+
+  def start_link(_), do: GenServer.start_link(__MODULE__, [], name: __MODULE__)
+
+  @impl true
+  def init(_), do: {:ok, true}
+
+  @impl true
+  def handle_call({:event, event, measurements, metadata, metrics}, _, _) do
+    Db.write_event(event, measurements, metadata, metrics)
```

Comment:
Why the move to run `Db.write_event/4` in the server and not in the telemetry handler?